### PR TITLE
Add Container View Icon and a View

### PR DESCRIPTION
**Pull request by hyunsik-yoon**
This adds container view icon.
Pressing the icon will bring a view for one-vscode.

parent issue: #330

ONE-vscode-DCO-1.0-Signed-off-by: Hyun Sik Yoon <hyunsik.yoon.1024@gmail.com>

### CODE REVIEW

**Comment by hyunsik-yoon** [comment]
1) icon was copied and modified from the logo of ONE project.
2) container icon
image
3) a view for one-vscode after clicking container icon
image

**Review by YongseopKim — DISMISSED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -21,6 +21,24 @@
   ],
   "main": "./out/extension.js",
   "contributes": {
+    "viewsContainers": {
+      "activitybar": [
+        {
+          "id": "ONE-Compiler",
+          "title": "ONE Compiler",
```
Comment (llFreetimell) [inline package.json:28]: It's only my preference, but how about just `ONE`? :)

Comment (hyunsik-yoon) [reply package.json]: Fixed.

Comment (hyunsik-yoon) [reply package.json]: image

**Review by llFreetimell — APPROVED** [review]
LGTM

**Review by YongseopKim — APPROVED** [review]
LGTM

**Pull request merged**